apps/guide/src/content/04-slash-commands/03-parsing-options.mdx

```diff
@@ -0,0 +1,148 @@
+---
+title: Parsing options
+category: Slash commands
+---
+
+# Parsing options
+
+In this section, you'll learn how to access the submitted values of a command interaction's options. Consider the following _`ban`_ command example with two options:
+
+<CH.Code>
+
+```js
+import { ApplicationCommandOptionType, PermissionFlagsBits } from 'discord.js';
+
+export const data = {
+	name: 'ban',
+	description: 'Select a member and ban them.',
+	options: [
+		{
+			name: 'target',
+			type: ApplicationCommandOptionType.User,
+			description: 'The member to ban',
+			required: true,
+		},
+		{
+			name: 'reason',
+			type: ApplicationCommandOptionType.String,
+			description: 'The reason for banning',
+		},
+	],
+};
+```
+
+</CH.Code>
+
+In the execute method, you can retrieve the value of these two options from the <DocsLink type="class" parent="CommandInteractionOptionResolver" /> as shown below:
+
+<CH.Code>
+
+```js
+export async function execute(interaction) {
+	const target = interaction.options.getUser('target');
+	const reason = interaction.options.getString('reason') ?? 'No reason provided';
+
+	await interaction.reply(`Banning ${target.username} for reason: ${reason}`);
+	await interaction.guild.members.ban(target);
+}
+```
+
+</CH.Code>
+
+Since _`reason`_ isn't a required option, the example above uses the _`??`_ [nullish coalescing operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator) to set a default value in case the user does not supply a value for _`reason`_.
+
+If the target user is still in the guild where the command is being run, you can also use <DocsLink type="class" parent="CommandInteractionOptionResolver" symbol="getMember" brackets /> on the `_target_` option to get their <DocsLink type="class" parent="GuildMember" /> object.
+
+<Alert title="Tip" type="success">
+If you want the Snowflake of a structure instead, grab the option via <DocsLink type="class" parent="CommandInteractionOptionResolver" symbol="get" brackets /> and access the Snowflake via the _`value`_ property. Note that you should use _`const { value: name } = ...`_ here to [destructure and rename](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment) the value obtained from the <DocsLink type="typedef" parent="CommandInteractionOption" /> structure to avoid identifier name conflicts.
+</Alert>
+
+In the same way as the above examples, you can get values of any type using the corresponding <DocsLink type="class" parent="CommandInteractionOptionResolver" /> \_`get_____()`_ method. _`String`_, _`Integer`_, _`Number`_ and _`Boolean`_ options all provide the respective primitive types, while _`User`_, _`Channel`_, _`Role`_, and _`Mentionable`\_ options will provide either the respective discord.js class instance if your application has a bot user in the guild or a raw API structure for commands-only deployments.
```

Comment:
that looks really, really odd in the actual guide 
![image](https://github.com/discordjs/discord.js/assets/26532370/6c7c3e09-9416-4144-a5df-99f0378e54ec)